.github/scripts/release-tags.js

```diff
@@ -55,8 +55,8 @@ function getDomain(context) {
 }
 
 function getReleaseTag(domain, app, context) {
-  const semver = context.payload.release.tag_name.replace('v', '')
-  if (semver.match(/^[0-9]+\.[0-9]+\.[0-9]+$/) === false) {
+  const semver = context.payload.release.tag_name
+  if (semver.match(/^v[0-9]+\.[0-9]+\.[0-9]+$/) === null) {
```

Comment:
The `v` is replaced previously, but change keeps the v? This might not be intentional?

Reply:
Also ideally `v` should not be kept for docker release tag

Reply:
If that is the case, will do the regex check as suggested by @jingyi2811 first, then return the version with the `v` removed

Reply:
Okay, I was misled by the `replace v with empty string` code.

Reply:
Why don't remove v from the regular expression?

Reply:
I fell like I probably misunderstanding something

Reply:
Either way it should work as the initial replace function implementation only replaces the first v (as mentioned in the previous thread). https://github.com/JellyfishSDK/jellyfish/commit/69cb57d7b487972f230d058449422fb854e083c3 will check the expression before replacing the `v` which works fine too. Actually, from my understanding it will prevent 1 additional string scenario which is e.g. `1.1.1` (initial semver without the v) from passing the regex condition